bob/mkosi.extra/etc/systemd/system/lighthouse-log-forwarder.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Lighthouse Log Forwarder for Searcher Access
+After=lighthouse.service
+Requires=lighthouse.service
+
+[Service]
+Type=simple
+User=lighthouse-logs
+Group=lighthouse-logs
+ExecStartPre=/bin/mkdir -p /run/lighthouse_logs
+ExecStartPre=/bin/chown lighthouse-logs:lighthouse-logs /run/lighthouse_logs
+ExecStart=/bin/sh -c "journalctl -f -u lighthouse.service --no-hostname --output=short-iso | tee /run/lighthouse_logs/current.log"
```

Comment:
I think a better way to do this would be to just run `tee` from within the lighthouse service itself rather than starting a new service. also, can we keep everything as just `lighthouse` user rather than making a new user?